docs/contribute-to-romcal.md

```diff
@@ -125,3 +125,51 @@ _pass the `-t` flag and the path of the file_
 ## Coverage reporting
 
 A coverage report is generated for every core branch build and published to [gh-pages]().
+
+## Committing changes
+
+### Message Format
+
+When you are ready to commit your changes, your commit message should follow the conventional commit format
+
+General format/terminology:
+
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
+```
+
+Here are some guardrails to follow:
```

Comment:
I’d probably replace `guardrails` with a simpler word, as the romcal contributors have ranging English skills, therefore, we should try to use simple language.

Just a note: I have never heard about this word before. :wink:

Maybe `rules`? 🤔

Reply:
```suggestion
Here are some rules to follow:
```